GAE/src/locale/en.properties

```diff
@@ -69,6 +72,7 @@ Before\ you\ save\ or\ publish\ a\ survey,\ you\ have\ to\ give\ the\ survey\ a\
 Before\ you\ save\ or\ publish\ a\ survey,\ you\ have\ to\ select\ a\ survey\ type = Before you save or publish a survey, you have to select a survey type
 Build\ chart = Build chart
 Bulk\ upload\ data = Bulk upload data
+By\ repeating\ this\ question\ group\ you\ can\ ask\ questions\ in\ this\ group\ multiple\ times,\ e.g.\ when\ you\ want\ to\ ask\ the\ same\ questions\ to\ each\ member\ of\ a\ household.\ You\ will\ decide\ on\ the\ number\ of\ repetitions\ when\ capturing\ the\ answers\ in\ the\ Akvo\ Flow\ app.\ You\ cannot\ make\ a\ question\ outside\ of\ this\ group\ dependent\ on\ a\ question\ inside\ the\ group. = By repeating this question group you can ask questions in this group multiple times, e.g. when you want to ask the same questions to each member of a household. You will decide on the number of repetitions when capturing the answers in the Akvo Flow app. You cannot make a question outside of this group dependent on a question inside the group.
```

Comment:
I am trying to understand the logic behind it: so on the left is the key which has the spaces escaped with "\", correct? In that case shouldn't the keys be shorter and more "explanotary" of where they are used, say for example "title\ left\ tab" or "question\ help\ description".

Reply:
in the en.properties we have the exact string. The one on the left is the key and thats why its escaped, and the one on the right is the string.